Draft v1.0/Specification.md

```diff
@@ -238,9 +244,39 @@ The following table gives guidelines for how to comply with the requirements in
+| STATIC_6 |  Metadata in the file property information such as author, title, category etc. can be misleading if i.e. the document was reused as a template. You may consider removing this information from the file to not mislead any future user of the data.<br><br>Metadata can be saved in a sidecar file or in an associated table, to document their existence before removal. |
+| STATIC_7 | The static file representation of the cell hyperlinks should be shared with the file created in STATIC_5 and have a fourth column appended in row 1 with the following information:<ul><li>*Column 4:* Cell hyperlink</li></ul> |
 
-## 6 Example Spreadsheets Software
+## 6 Metadata Preservation Scheme
+
+This specification defines a preservation scheme for the metadata in a spreadsheet or metadata related to altering the content of a spreadsheet.
+
+The metadata may be preserved in a sidecar file for each spreadsheet or preserved in a database table with metadata for multiple spreadsheets.
+
+###  6.1 Metadata Fields
+
+The following table describes which metadata should be preserved and in which circumstances.
+
+**Table 9. Metadata that may be preserved.**
+
+| IDs | Name | Description | Data type | Circumstance |
+| --- | --- | --- | --- | --- |
+| ODS_10, OOXML_12, CSV_5, STATIC_6 | Filename | The filename of the spreadsheet. | String | Every time |
+| ODS_10, OOXML_12, CSV_5, STATIC_6 | Creator | The creator of the spreadsheet in the file properties. | String | If value exist |
+| ODS_10, OOXML_12, CSV_5, STATIC_6 | Creation date | The creation date of the spreadsheet in the file properties. | DateTime | Every time |
+| ODS_10, OOXML_12, CSV_5, STATIC_6 | Title | The title of the spreadsheet in the file properties. | String | If value exist |
+| ODS_10, OOXML_12, CSV_5, STATIC_6 | Category | The categories of the spreadsheet in the file properties. | String(s) | If value exist |
+| CSV_3, STATIC_3 | Sheets | The names of all sheets. | String(s) | Every time |
+| ODS_4, OOXML_5 | External references | Any external reference to data e.g. data source, cell reference, RealTimeData. | String(s)<br><br>Cell position attribute | If removed |
+| ODS_5, OOXML_6 | Embedded objects | Any embedded object that has been removed. | String(s)<br><br>Cell position attribute | If removed |
+| STATIC_6 | Formulas | Any formulas that has been removed from the spreadsheet. | String(s)<br><br>Cell position attribute | Every time |
+| ODS_7, OOXML_8 | Macros | Any macros that has been removed from the spreadsheet. | String(s) | If removed |
+| ODS_11, OOXML_13, STATIC_6 | Cell hyperlinks | Any hyperlinks to websites in any cells. | String(s)<br><br>Cell position attribute | Every time |
+
+### 6.2 Example Metadata File
+You may find an example of a metadata preservation sidecar file created in XML [here](/v1.0/Example%20Metadata%20File.xml).
+
+## 7 Example Spreadsheets Software
```

Comment:
The more I think about this, the less I'm convinced it's a great idea:

1. It just becomes the half-way house for things that there seems to be a desire to remove from the document.
2. We're in danger of inventing another metadata format where others exist for this kind of thing:
  - PREMIS for preservation events, and what was removed
  - various metadata formats for creator, title, etc.
  - METS as an information package standard to capture structural info, such as the relationships between the original sheet structure and CSV file (structMap)
  - ODF/OO XML for spreadsheet artefacts like macros and formulas
3. Remember this software doesn't redact files, it validates against the original specification and then enforces compliance with further rules/reports preservation risks.

Reply:
Your validator is one piece of the software required to make this specification function e.g. the specification also has policies about OOXML and we don't create a validator for that format now.

We may consider removing the Metadata Example File (the sidecar) and solely make recommendations of which metadata to preserve in the context of spreadsheets and under which circumstances. Then we may write it is up to the organisation to initiate measures on how to capture the metadata.

Reply:
From the perspective of significant properties, I would like to propose the following:

Spreadsheets can contain metadata such as (original) filename, title, author and  creation date, or have associated metadata about the context of or events related to the spreadsheet, Spreadsheet metadata SHOULD BE preserved when considered a significant property. It is up to your organization to determine the best approach and metadata standard for preserving such metadata. Metadata that is considered an artifact MAY BE removed.